pkg/cmd/kafka/create/create.go

```diff
@@ -116,12 +122,18 @@ func NewCreateCommand(f *factory.Factory) *cobra.Command {
 	flags.AddOutput(&opts.outputFormat)
 	flags.BoolVar(&opts.autoUse, "use", true, opts.localizer.MustLocalize("kafka.create.flag.autoUse.description"))
 	flags.BoolVarP(&opts.wait, "wait", "w", false, opts.localizer.MustLocalize("kafka.create.flag.wait.description"))
-	flags.AddBypassTermsCheck(&opts.bypassTermsCheck)
+	flags.AddBypassTermsCheck(&opts.bypassAmsCheck)
```

Comment:
This flag was introduced for kas-installer to skip all AMS specific elements. 
Logic for validating provider and AZ will not be valid for Kas-Installer (same as terms and conditions).

Instead of introducing another flag I have reused the same to skip all AMS validation for create command. 
I considered renaming flag in separate PR and send message to team that uses it.